module/purpose/Striker/data/config/Striker.yaml

```diff
@@ -2,4 +2,4 @@
 log_level: INFO
 
 # Position on field to walk to (x,y,theta)
-ready_position: [4, 0, -3.14]
+ready_position: [1.2, 0, -3.14]
```

Comment:
Honestly I might consider putting a further comment as to what this means for future people changing this, is -3.14 towards the enemy goal? towards ours?
What is the coordinate system?
Either reference another coordinate system (e.g. make this variable rPFf or something) or some other way describe it.

Also once again if this is an expression you can just put `pi` rather than `3.14`

Reply:
this isn't even relevant for this PR :weary: